dist/index.js

```diff
@@ -580,32 +580,35 @@ function handleMain(inputs, client) {
                 case 0: return [4, (0, changes_1.getChangesInPush)(inputs.defaultBranch)];
                 case 1:
                     changes = _a.sent();
-                    if (!changes.additions.includes(inputs.haltFile)) return [3, 3];
```

Comment:
When you don't notice you're looking at the minified file
![](https://thumbs.gfycat.com/TartNecessaryCleanerwrasse-size_restricted.gif)

Reply:
I did the same thing!